http-server/src/test/java/io/airlift/http/server/TestDelimitedRequestLog.java

```diff
@@ -143,7 +141,7 @@ public void testWriteLog()
 
         String actual = Files.toString(file, UTF_8);
         String expected = String.format("%s\t%s\t%s\t%s\t%s\t%s\t%d\t%d\t%d\t%d\t%s\n",
-                isoFormatter.print(timestamp),
+                ISO_FORMATTER.format(Instant.ofEpochMilli(timestamp)),
```

Comment:
I checked the `http-request.log` from the sample server:
```
2016-05-10T12:26:31.589-07:00	0:0:0:0:0:0:0:1	GET	/	null	Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/50.0.2661.94 Safari/537.36	200	0	52	1	null
2017-01-25T17:15:49.619-08:00	0:0:0:0:0:0:0:1	GET	/	null	Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.95 Safari/537.36	404	0	232	37	2cef4ec0434a441a84e38d94674d43750000000000
```
It's unfortunate the tests don't compare against a hard-coded output string.